test/unit/acknowledgement/update-acknowledgement.test.js

```diff
@@ -329,45 +326,13 @@ describe('update acknowledgement', () => {
     expect(processInvalid).toHaveBeenCalledTimes(1)
   })
 
-  test('should call processInvalid with getPaymentRequest schemeId, paymentRequestId and frn as well as message when an unsuccessful ack object is given', async () => {
+  test('should call processInvalid with getPaymentRequest schemeId, paymentRequestId, frn and acknowledgement when an unsuccessful ack object is given', async () => {
```

Comment:
same comment on test names as in `processInvalid`, should save you some confusion

Reply:
As in the test names are wrong or to shorten them as they're repeating similar information. I'm unsure which comment you mean

Reply:
Yup, shorten.

This should just be `'should call processInvalid'`